app/models/user.php

```diff
@@ -82,5 +81,136 @@ public function login(LoginInfo $login_info)
             throw new RecordNotFoundException('No Record Found');
         }
         Session::set('username', $this->username);
+        Session::set('id', $row['id']);
+    }
+
+    public static function getAll()
+    {
+        $user = array();
+        $db = DB::conn();
+        $rows = $db->rows("SELECT * FROM user WHERE id = ?", array(Session::get('id')));
+
+        foreach ($rows as $row) {
+            $user[] = new self($row);
+        }
+        return $user;
+    }
+
+    public static function getComments()
+    {
+        $comments = array();
+        $db = DB::conn();
+        $rows = $db->rows("SELECT * FROM comment 
+                    WHERE username = ?", array(Session::get('username')));
+
+        foreach ($rows as $row) {
+            $comments[] = new self($row);
+        }
+        return $comments;
+    }
+
+    public function getId()
+    {
+        $db = DB::conn();
+        $id = $db->row("SELECT id FROM user WHERE id = ?", array(Session::get('id')));
```

Comment:
in model, we usually pass the parameters as arguments in the function
maybe you can refactor this? like:
```php
public static function getById($id)
```

then you can just call it in controller like:
```php
User::getById(Session::get('id'));
```